scylla-cdc-base/src/main/java/com/scylladb/cdc/model/worker/Task.java

```diff
@@ -26,4 +27,24 @@ public Task updateState(ChangeId lastSeenChangeId) {
         Preconditions.checkNotNull(lastSeenChangeId);
         return new Task(id, streams, state.update(lastSeenChangeId));
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
```

Comment:
wouldn't `!(o instanceof Task)` be more concise?

Reply:
Yes and in this case, it is correct to do so (final class).

In general, if this class was not final, using `instanceof` would break symmetry of `equals` when comparing with subclass objects. See full context here: https://youtrack.jetbrains.com/issue/IDEA-7692

Reply:
Fixed in the amended commit.